cluster-scope/base/user.openshift.io/groups/prometheus-anomaly-detector/group.yaml

```diff
@@ -2,4 +2,7 @@ apiVersion: user.openshift.io/v1
 kind: Group
 metadata:
     name: prometheus-anomaly-detector
-users: []
+users:
+    # FIXME 1 account missing
```

Comment:
these two users should be fine

Reply:
But can you get the github username for the 3rd person anyways :smile: ?